### PR TITLE
Add Display movies js file and add get data from api and styles

### PR DESCRIPTION
**Pull request by DhuruvSingh**
# [Display list of items on the Homepage](https://github.com/DhuruvSingh/CapstoneJS/issues/15)

- :ballot_box_with_check: Adding JavaScript modules folder

- :ballot_box_with_check: Adding 'display.js'

- :ballot_box_with_check: Adding the functions to get data from the API & display it

> **Please Review and Give your valuable feedback**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="https://kit.fontawesome.com/e4fb871c71.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
```
Comment (Peter1907) [inline src/index.html]: You don't have to import the styles as it is compiled with webpack through the JavaScript

File: src/index.html [hunk 2]
```diff
@@ -31,8 +32,9 @@ <h2>Available Shows</h2>
     <div class="screen hidden"></div>
 
     <footer>
-        Created by <a href="https://github.com/Peter1907">Peter Beshara</a>&
+        Created by <a href="https://github.com/Peter1907">Peter Beshara </a>&
         <a href="https://github.com/DhuruvSingh">Dhuruv Singh</a>
     </footer>
+    <script src="./index.js" type="modules"></script>
```
Comment (Peter1907) [inline src/index.html]: Also you don't have to include a script tag at the end

**Review by Peter1907 — APPROVED** [review]
## Project Approved 👍 

Hi @DhuruvSingh  ,

This project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Highlights :1st_place_medal:
:heavy_check_mark: Linters passing
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following the requirements
:heavy_check_mark: Using Gitflow
:heavy_check_mark: Correctly using the API

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/index.html (hunk 1 above)
Comment (DhuruvSingh) [reply src/index.html]: sorry for that i just added this for locally test

File: src/index.html (hunk 2 above)
Comment (DhuruvSingh) [reply src/index.html]: ok ill remove it

**Comment by DhuruvSingh** [comment]
Thankyou For your valuable feedback @Peter1907

File: src/index.html (hunk 2 above)
Comment (DhuruvSingh) [reply src/index.html:38]: ok ill remove it

**Pull request merged**